ceph/cephfs/auth/openshift-clusterrole.yaml

```diff
@@ -0,0 +1,20 @@
+kind: ClusterRole
+apiVersion: v1
+metadata:
+  name: cephfs-provisioner-runner
+rules:
+  - apiGroups: [""]
+    resources: ["persistentvolumes"]
+    verbs: ["get", "list", "watch", "create", "delete"]
+  - apiGroups: [""]
+    resources: ["persistentvolumeclaims"]
+    verbs: ["get", "list", "watch", "update"]
+  - apiGroups: ["storage.k8s.io"]
+    resources: ["storageclasses"]
+    verbs: ["get", "list", "watch"]
+  - apiGroups: [""]
+    resources: ["events"]
+    verbs: ["list", "watch", "create", "update", "patch"]
+  - apiGroups: [""]
+    resources: ["secrets"]
+    verbs: ["get", "create", "delete"]
```

Comment:
well, new provisioners like storageos and provisioners like azure that *are* copying admin credentials into namespaces are forcing revisiting the issue, which is good. Fixing the data model so that provisioners can place secrets (either admin or per-volume) into a separate namespace, reference the secret name/namespace in the PV, and only have their provisioner able to read/write secrets in that namespace seems like a good path forward.